task/gcloud/0.3/README.md

```diff
@@ -0,0 +1,165 @@
+# `gcloud`
+
+This task performs operations on Google Cloud Platform resources using `gcloud`.
+
+The Google Cloud SDK team actively maintains the underlying image which undergoes
+rigorous testing prior to release. Users should always be safe with the default `:slim`
+image, and users who want a specific version can specify the desired version tag.
+
+For details on the underlying image, see https://github.com/GoogleCloudPlatform/cloud-sdk-docker
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/gcloud/0.3/gcloud.yaml
+```
+
+## Parameters
+
+* **gcloud-image**: `gcloud` CLI container image to run this task.
+
+  _default_: `gcr.io/google.com/cloudsdktool/cloud-sdk`
+
+  You can override this param with a custom `gcloud` build if needed. If doing
+  so, you likely want to override _version_ as well.
+
+* **version**: `gcloud` version to use (_default:_ slim)
+
+  You can use a specific version of the `gcloud` CLI by specifying
+  a tag; available tags can be seen at https://gcr.io/google.com/cloudsdktool/cloud-sdk
+
+  The default image (`:slim`) does not include any optional gcloud components.
+  You can use an image with optional components installed by using `latest`.
+
+* **ARGS**: The arguments to pass to `gcloud` CLI.  _default_: `["help"]`
+
+* **SCRIPT**: Script to execute. _default_: `gcloud $@`
+
+  You can run other installed commands by overriding the default; there is
+  sample code in the [tests](tests) directory that does this.
+
+## Platforms
+
+The Task can be run on `linux/amd64` platform.
+
+## Usage
+
+### Authorizing `gcloud` commands
```

Comment:
Thank you for the super detailed docs! This is one area that trips up a lot of users!